AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -429,6 +429,9 @@ pornorips.com#?#h1 > span:contains(AddBlocker)
 cablegratis.online##.alert-danger
 deepfun.net##div[id^="fwin_banadblock"]
 feber.se##div.overlayBox[id^="202"]
+@@||escripts.club/wutime-adblocktracker/
+escripts.club$$script[tag-content="wutime-adblocktracker"][max-length="150000"]
+escripts.club#%#//scriptlet("abort-current-inline-script", "$", "adblock")
```

Comment:
It doesn't work.

Reply:
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/107535467-cf964200-6bd1-11eb-8f18-bcedd0d5a367.png)</details><br/>

Reply:
Have you tested it with account? Still doesn't work.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/107606190-61d82d80-6c46-11eb-97dd-99616e2173e5.png)

</details><br/>